ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/spectrads3/AllocateJobChunkSpectraS3Response.java

```diff
@@ -18,8 +18,8 @@
 
 import com.spectralogic.ds3client.networking.WebResponse;
 import java.io.IOException;
-import java.io.InputStream;
 import com.spectralogic.ds3client.models.Objects;
+import java.io.InputStream;
```

Comment:
I moved the import from the template into the generator to stop it being added when not necessary. This also caused the order of appearance to change.